static/src/stylesheets/module/identity/_header.garnett.scss

```diff
@@ -0,0 +1,132 @@
+$gutter: 6px;
+$height: $identity-header-height - ($gutter * 2);
+
+$nav-background-colour: mix($sport-garnett-main-1, $neutral-7, 4%); // BG of the nav & menu
```

Comment:
Why don't you just add an import at the top of this file for https://github.com/guardian/frontend/blob/master/static/src/stylesheets/layout/garnett-header/_variables.scss ?

Reply:
I did not know i could do that. thanks!!